committing.rst

```diff
@@ -347,6 +337,8 @@ will still be issued but the changeset will be applied anyway::
 Clones Setup
 ------------
 
+// TODO is that still needed?
```

Comment:
I don't think it's worth keeping here. I haven't looked at the PR yet (started to read the comments first), but I think as long as we explain how to set up a different remote (e.g. `origin` and `upstream` for your fork and python/cpython, respectively) then that should cover it.